Web Development/README.md

```diff
@@ -104,6 +104,10 @@ This category is a curated collection of valuable web development resources cont
     <td><a href="https://www.youtube.com/watch?v=bMknfKXIFA8">React Course - Beginner's Tutorial for React JavaScript Library [2022]</a></td>
     <td>React course for beginners. It simplifies and demystifies react to any beginner who is willing and eager to learn.</td>
   </tr>
+  <tr>
```

Comment:
This is not suitable to the web development under tutorial or courses, instead you can put it in the [Category Section](https://github.com/jfmartinz/ResourceHub/tree/main/UI-UX%20Design) under [Tutorial or Courses](https://github.com/jfmartinz/ResourceHub/tree/main/UI-UX%20Design#tutorials-or-courses). After these change, I will merged your pr, thank you!